test/over_react/util/map_util_test.dart

```diff
@@ -253,5 +254,54 @@ main() {
         expect(newStyleFromProps(domProps()), equals({}));
       });
     });
+
+    group('getBackingMap()', () {
+      group('when passed in a props object', () {
+        JsBackedMap testMap = JsBackedMap.from({'id': 'test'});
+
+        var props = getBackingMap(TestProps(testMap));
+
+        test('returns a map', () {
+          expect(props, TypeMatcher<Map>());
+        });
+
+        test('returns the correct custom map values', () {
+          expect(props.containsKey('id'), isTrue);
+          expect(props['id'], 'test');
```

Comment:
FYI you can just do
```dart
expect(props, testMap);
```
and it will perform map equality

Reply:
Nice, I wasn't sure because map comparisons are a little more complex than they seem, so good to know that works.